packages/orchestrator-ui-components/src/components/WfoSubscription/WfoInUseByRelations.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+
+import { useTranslations } from 'next-intl';
+import Link from 'next/link';
+
+import {
+    WfoError,
+    WfoFirstPartUUID,
+    WfoKeyValueTable,
+    WfoLoading,
+} from '@/components';
+import type { WfoKeyValueTableDataType } from '@/components';
+import { useWithOrchestratorTheme } from '@/hooks';
+import { useGetInUseByRelationDetailsQuery } from '@/rtk';
+import { InUseByRelation, InUseByRelationDetail } from '@/types';
+
+import { getStyles } from './styles';
+
+interface WfoInUseByRelationsProps {
+    inUseByRelations: InUseByRelation[];
+}
+
+export const WfoInUseByRelations = ({
+    inUseByRelations,
+}: WfoInUseByRelationsProps) => {
+    const t = useTranslations('subscriptions.detail');
+    const { inUseByRelationDetailsStyle } = useWithOrchestratorTheme(getStyles);
+    const subscriptionIds = inUseByRelations
```

Comment:
This is input for the query to the backend, it might be better to move the mapper with `|` to the data fetch part of code. In the end we just want to provide a list (array) of id's to fetch data for

Reply:
There is a ticket to improve this and resolve all of it in the backend: https://github.com/workfloworchestrator/orchestrator-ui-library/issues/1031